lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -571,6 +575,10 @@ value).
 
   my $html = $c->render_to_string('reminder', layout => 'mail');
 
+Note that when searching for a file based layout, the C<layout> helper will
```

Comment:
I'd say something like 'when a layout is set' since you can set it via a helper or a stash key.